### PR TITLE
Api

### PR DESCRIPTION
**Pull request by joshualang**
Implemented a firestore database with api in node.js

### CODE REVIEW

File: server/firebase.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+//   hepatitisB: "Hepatitis B",
+//   pneumokokken: "Pneumokokken",
+//   meningokokkenC: "Meningokokken C",
+//   masern: "Masern",
+//   mumps: "mumps",
+//   varizellen: "Varizellen",
+//   hpv: "HPV",
+//   herpesZoster: "Herpes zoster",
+//   influenza: "Influenza"
+// }
```
Comment (dvtdyn) [inline server/firebase.js]: Remove comments, if you don't need them

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+})
+
+let db = admin.firestore()
+
+const express = require('express')
+const uid = require('uid')
+const cors = require('cors')
+
+const server = express()
+const port = 3334
```
Comment (dvtdyn) [inline server/server.js]: May implement a general port

File: src/VaccinationForm.js [hunk 3]
```diff
@@ -1,94 +1,157 @@
...
-          <input
+          <Input
             id="vaccinationSticker"
             name="vaccinationSticker"
-            onInput={event => setForm({ ...form, sticker: event.target.value })}
+            onInput={event =>
+              setForm({
+                ...form,
+                sticker: event.target.value,
+                validSticker: event.target.value !== '',
```
Comment (dvtdyn) [inline src/VaccinationForm.js]: Remove !== ' '

File: src/hooks/useLoadingEffect.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+  useEffect(() => {
+    getData().then(loadedData => {
+      setData(loadedData)
+      setIsLoading(false)
+    })
+  }, [])
+
+  return { data, isLoading }
+}
```
Comment (dvtdyn) [inline src/hooks/useLoadingEffect.js]: fresh code $$$$

File: src/services.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+export function getData() {
+  return fetch('http://localhost:3334/api/LA').then(res => res.json())
+  //.catch(err => console.log("--->", err))
```
Comment (dvtdyn) [inline src/services.js]: Remove comment

File: src/hooks/useLoadingEffect.js (hunk 4 above)
Comment (joshualang) [reply src/hooks/useLoadingEffect.js]: thanks!

**Pull request merged**